### PR TITLE
Simplify usage of resource namespace and fix tests correspondingly

### PR DESCRIPTION
**Pull request by bnsblue**
Related PRs
https://github.com/lyft/flyteplugins/pull/44
https://github.com/lyft/flyteplugins-private/pull/12

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -57,7 +57,8 @@ required = [
 
 [[constraint]]
   name = "github.com/lyft/flyteplugins"
-  version = "^0.2.5"
+  # version = "^0.2.5"
+  branch = "simpler-hive-plugin-init"
```
Comment (katrogan) [inline Gopkg.toml]: use a release version?

Comment (bnsblue) [reply Gopkg.toml]: https://github.com/lyft/flyteplugins/pull/44

Comment (bnsblue) [reply Gopkg.toml]: I'll update it after that PR is merged

File: pkg/controller/nodes/task/handler.go [hunk 2]
```diff
@@ -176,10 +175,12 @@ func (t *Handler) Setup(ctx context.Context, sCtx handler.SetupContext) error {
...
 	for _, p := range enabledPlugins {
-		// rn = create a new resource negotiator proxy for each plugin
-		sCtxFinal := newNameSpacedSetupCtx(tSCtx, newResourceManagerBuilder.ResourceRegistrar(pluginCore.ResourceNamespace(p.ID)))
+		// create a new resource registrar proxy for each plugin, and pass it into the plugin's LoadPlugin() via a setup context
+		logger.Warningf(ctx, "plugin ID = %v\n", p.ID)
+		sCtxFinal := newNameSpacedSetupCtx(tSCtx, newResourceManagerBuilder.GetResourceRegistrar(pluginCore.ResourceNamespace(p.ID)))
+		logger.Warningf(ctx, "plugin ID = %v\n", p.ID)
 		// sCtxFinal := newNSSetupCtx(tSCtx)
-		// tSCtx.resourceNegotiator = tSCtx.ResourceRegistrar().ResourceRegistrar(pluginCore.ResourceNamespace(p.ID))
+		// tSCtx.resourceNegotiator = tSCtx.GetResourceRegistrar().GetResourceRegistrar(pluginCore.ResourceNamespace(p.ID))
```
Comment (katrogan) [inline pkg/controller/nodes/task/handler.go]: why is this commented out/updated?

File: pkg/controller/nodes/task/resourcemanager/redis_client.go [hunk 3]
```diff
@@ -9,13 +9,47 @@ import (
...
 
-func NewRedisClient(ctx context.Context, config config.RedisConfig) (*redis.Client, error) {
-	client := redis.NewClient(&redis.Options{
-		Addr:       config.HostPath,
-		Password:   config.HostKey,
-		DB:         0, // use default DB
-		MaxRetries: config.MaxRetries,
-	})
+type RedisClient interface {
+	SCard(string) *redis.IntCmd
```
Comment (katrogan) [inline pkg/controller/nodes/task/resourcemanager/redis_client.go]: Can you add comments for these public methods?

**Pull request merged**